openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx

```diff
@@ -417,7 +419,11 @@ export const getDataLabel = (
     <span
       className="tw-break-words description-text tw-self-center"
       data-testid="lineage-entity">
-      {label}
+      {type === 'table'
+        ? databaseName
+          ? `${databaseName}/${label}`
```

Comment:
Should this be `.` instead of `/`

Reply:
@mithmatt, we are using `/` in suggestions as well so we will use `/` everywhere.

Reply:
@Sachin-chaurasiya I think its better "." as we are using that as a separation in FQN. It will be consistent with our APIs as well